pkg/restore/restore_test.go

```diff
@@ -2113,6 +2112,130 @@ func TestRestorePersistentVolumes(t *testing.T) {
 	}
 }
 
+type fakeResticRestorerFactory struct {
+	restorer         *resticmocks.Restorer
+	podVolumeBackups []*velerov1api.PodVolumeBackup
```

Comment:
I don't think the `podVolumeBackups` field is used - is it? let's delete if not.